### PR TITLE
Add multi value metrics support

### LINKED ISSUE #20: Support multivalue metric names
#2 added support for multivalue aggregates, now we need support for multivalue metric names

### PR DESCRIPTION
**Pull request by kinnarr**
Fixes #20

### CODE REVIEW

File: metricq_grafana/amqp.py [hunk 1]
```diff
@@ -18,14 +18,20 @@ async def get_history_data(app, request):
...
     targets = []
     for target_dict in request["targets"]:
-        targets.append(
-            Target(
-                metric=target_dict["metric"],
-                name=target_dict.get("name", None),
-                functions=list(parse_functions(target_dict)),
-                scaling_factor=float(target_dict.get("scaling_factor", "1")),
+        metrics = [target_dict["metric"]]
+        if "(" in target_dict["metric"] and ")" in target_dict["metric"]:
```
Comment (tilsche) [inline metricq_grafana/amqp.py:22]:
```suggestion
        # guess if this is a pattern (regex) to expand and 
        if "(" in target_dict["metric"] and ")" in target_dict["metric"]:
```

**Review by tilsche — CHANGES_REQUESTED** [review]
Just a tad of inline-documentation what we do and we should be good to go.

**Comment by tilsche** [comment]
Oh, and what happens if the manager returns an empty list? Maybe we should capture that somehow - can we even?

**Comment by kinnarr** [comment]
If the manager returns an empty list, it also returns an empty, which results in "no data" being displayed in grafana.

**Comment by tilsche** [comment]
> If the manager returns an empty list, it also returns an empty, which results in "no data" being displayed in grafana.

I can live with that.

**Pull request merged**